ckan/templates-midnight-blue/package/base_form_page.html

```diff
@@ -26,6 +26,7 @@
     {% block page_header %}{% endblock %}
     <div class="module-content">
       {% block primary_content_inner %}
+        <h1>{% block form_title %}{% endblock %}</h1>
```

Comment:
This is a new block and pages from existing themes might not override it.

Assuming that an empty `h1` is worse than a missing `h1` (it creates a gap in the page, right?) we could wrap this in an `{% if %}` block that checks that there is `form_title` content defined.